### ISSUE TITLE
Implement new search

### ISSUE THREAD

**Issue by noahmanger** [18F/openFEC#432]
**Goal:** A beautiful new search results page.

- [x] Make final design edits

**Functionality (@msecret)**
- [x] Hide header search on home page
- [x] Implement "candidate / committee" select on search fields
- [x] Implement appropriate API calls and return the results
- [ ] Implement the two-year period selection on the results page
- [ ] The search query should be passed through and displayed on both the "name includes:" field as well as the search bar in he header
- [x] Output all data necessary for each result

**Display (@noahmanger)**
- [ ] Style "candidate / committee" select (main and header)
- [ ] Style search results page
- [ ] Create sidebar boxes with the correct links

**On hold**
I think we can wait on:
- [ ] Implement "sort by" select with the option

**Comment by rjmajma** [comment]
#522 is important for this ticket. Don't ask me why though.

**Comment by rjmajma** [comment]
Hey @noahmanger, where is the mockup that Marco is going off of for this?

**Comment by noahmanger** [comment]
Oops, here.

On the home page: hide the header search and add the dropdown to choose between candidates and committees on the search.

![new search](https://cloud.githubusercontent.com/assets/1696495/7164712/2cf55a76-e370-11e4-8988-f218a731c445.png)

![new search - dropdown](https://cloud.githubusercontent.com/assets/1696495/7164720/421944d0-e370-11e4-805b-72903c4b0d14.png)

And then the search results page:
![new search results](https://cloud.githubusercontent.com/assets/1696495/7164725/4c6bf3d8-e370-11e4-8ccf-bc60f9d6f370.png)

And here's how the date range thing works:
![additional filters](https://cloud.githubusercontent.com/assets/1696495/7164731/5a18990a-e370-11e4-9e66-84866a34b368.png)
